awxkit/awxkit/cli/resource.py

```diff
@@ -121,12 +123,14 @@ def parse_resource(client, skip_deprecated=False):
             if k in ('dashboard',):
                 # the Dashboard API is deprecated and not supported
                 continue
-            aliases = []
-            if not skip_deprecated:
+
+            # argparse aliases are *only* supported in Python3 (not 2.7)
```

Comment:
I want to follow-up and see if I can come up with another way to support this in py2